pkg/driver_atwinc15x0/Makefile

```
@@ -0,0 +1,17 @@
+PKG_NAME=driver_atwinc15x0
+PKG_URL=https://github.com/arduino-libraries/WiFi101
+PKG_VERSION=3301d03f82e53f60e07434a9f07ca677d992d121
+PKG_LICENSE=LGPL-2.1
+
+include $(RIOTBASE)/pkg/pkg.mk
+
+all:
+	"$(MAKE)" MODULE=driver_atwinc15x0 \
+	          -C $(PKG_BUILDDIR)/src/driver/source \
+	          -f $(CURDIR)/Makefile.driver_atwinc15x0
+	"$(MAKE)" MODULE=driver_atwinc15x0_common \
+	          -C $(PKG_BUILDDIR)/src/common/source \
+	          -f $(CURDIR)/Makefile.driver_atwinc15x0
+	"$(MAKE)" MODULE=driver_atwinc15x0_spi_flash \
+	          -C $(PKG_BUILDDIR)/src/spi_flash/source \
+	          -f $(CURDIR)/Makefile.driver_atwinc15x0
```

Sorry to bother you with this. With the following change the build output will be a bit nicer (all `make -C xxx` are aligned).

```suggestion
	"$(MAKE)"  -C $(PKG_BUILDDIR)/src/driver/source -f $(CURDIR)/Makefile.driver_atwinc15x0 MODULE=driver_atwinc15x0
	"$(MAKE)" -C $(PKG_BUILDDIR)/src/common/source -f $(CURDIR)/Makefile.driver_atwinc15x0 MODULE=driver_atwinc15x0_common
	"$(MAKE)" -C $(PKG_BUILDDIR)/src/spi_flash/source -f $(CURDIR)/Makefile.driver_atwinc15x0 MODULE=driver_atwinc15x0_spi_flash
```

---

I made these line breaks by intention because of the maximum length of line of 100 characters.

---

Indeed